azure-pipelines.yml

```diff
@@ -112,16 +82,18 @@ stages:
   - stage: TESTS3
     dependsOn: BUILD
     displayName: Tests - Smoke
+    variables:
+      hmi-apim-svc-sub-key: stageDependencies.BUILD.Terraform.outputs['exportKeyVaultSecret.variableName']
     jobs:
-    - job:
+    - job: testsmoke
       pool:
         vmImage: 'windows-latest'
       steps:
       - task: Gradle@2
         inputs:
           gradleWrapperFile: 'gradlew'
           workingDirectory: '$(System.DefaultWorkingDirectory)'
-          tasks: clean smoke -DTEST_SUBSCRIPTION_KEY=$(hmi-apim-svc-sbox-sub-key) -DTEST_URL=https://hmi-apim-svc-sbox.azure-api.net -DTEST_HOST=hmi-apim-svc-sbox.azure-api.net
+          tasks: clean smoke -DTEST_SUBSCRIPTION_KEY=$(hmi-apim-sub-key) -DTEST_URL=https://hmi-apim-svc-sbox.azure-api.net -DTEST_HOST=hmi-apim-svc-sbox.azure-api.net
```

Comment:
shouldn't this one be test not sbox?

Reply:
@bugbare-reform - This is not my implementation or file...It came up in the rebase from master....Not part of this implementation.